core/src/main/scala/akka/kafka/KafkaConsumerActor.scala

```diff
@@ -254,11 +267,18 @@ private[kafka] class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V])
 
   def poll(): Unit = {
     val wakeupTask = context.system.scheduler.scheduleOnce(settings.wakeupTimeout) {
+      log.warning("KafkaConsumer poll is taking significantly longer ({}ms) to return from poll then the configured poll interval ({}ms). Waking up consumer to avoid thread starvation. Thread dump:", settings.wakeupTimeout.toMillis, pollTimeout().toMillis)
+      if (settings.wakeupDebug) {
+        // See what the KafkaConsumer is up to. Should we do just this dispatcher?
```

Comment:
good to dump as much as possible until we have this issue completely under control

Reply:
There's a heart beat thread that'll be useful to know if it is active too, so will leave this in